README.md

```diff
@@ -86,34 +86,76 @@ frontend   docker-entrypoint.sh /bin/ ...   Up
 web        ./manage.py runserver 0.0. ...   Up          0.0.0.0:8000->8000/tcp
 ```
 
+### Build the backend Docker image
+
+```sh
+make build
+```
+
+### Bring the backend Docker container up
+
+```sh
+make up
+```
+
+### Stop the backend Docker container
+
+```sh
+make down
+```
+	
 ### Run tests
 
 ```sh
-docker-compose exec -w /code/wagtail web python runtests.py
+make test
+```
+
+### Run tests for a specific file
+
+```sh
+make test file=wagtail.admin.tests.test_name.py
 ```
 
 ### Open a Django shell session
 
 ```sh
-docker-compose exec web python manage.py shell
```

Comment:
perhaps use both commands (make and direct one), in all examples?
some people may prefer the direct command

Reply:
A good idea for sure, I'm just curious how we would want to document it?
Something like this perhaps?

| Make command | Raw command                |
|--------------|----------------------------|
| `make build` | `docker-compose build web` |

Reply:
I agree, we should have documentation for both approaches.